app/src/main/java/com/example/criminalintent/data/local/CrimeDatabase.kt

```diff
@@ -1,11 +1,22 @@
 package com.example.criminalintent.data.local
 
+import androidx.room.AutoMigration
 import androidx.room.Database
 import androidx.room.RoomDatabase
 import androidx.room.TypeConverters
 import com.example.criminalintent.data.model.Crime
 
-@Database(entities = [Crime::class], version = 2)
+@Database(
+    entities = [Crime::class],
+    version = 3,
+    autoMigrations = [
```

Comment:
You need to specify migration from 1-2 as well. If the app is released with version 1, and the user installs version 3, it will not know how to upgrade to version 3.